.github/workflows/test-docker-image.yml

```diff
@@ -0,0 +1,37 @@
+name: Test Applciation Running
+
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Docker Compose Build
+      run: docker-compose build
+
+    - name: Docker Compose Up
+      run: docker-compose up -d
+
+    - name: Wait for Application to be ready
+      run: sleep 60 # Wait for 60 seconds for application to be ready
```

Comment:
On second thought, I think this is actually necessary but 60 seconds if def overboard, how about 10 seconds?

Reply:
Apparently, the port mapping is not necessarily exposed the same way on the GitHub Actions runner as on a local machine, so that's why it might not work. Might require a rethink